src/main/webapp/resources/js/pages/admin/routes.js

```diff
@@ -0,0 +1,10 @@
+/*
+ * This file contains the route names for the admin panel
+ */
+
+export const ADMIN = {
+  ADMIN: "admin",
+  STATISTICS: "statistics",
+  USERS: "users",
+  GROUPS: "groups"
```

Comment:
Rename to UserGroups: "user-groups" and update the key wherever it is used

Reply:
I am using these routes to change the URL. The old settings cog had /groups. Should it still be changed in this case? That means the path would then become admin/user-groups when the groups link is clicked